ci/ci_steps.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-ENVOY_BUILD_SHA=fc747b3c2fd49b1260484572071fe4194cd6824d
+ENVOY_BUILD_SHA=2867b337182c79ec5229a4261ecdadbf916f49ab4629dfea787cf76c91952dc4
```

Comment:
Thanks for the much needed pointer, I was thinking this was what was reported from the "docker images" command which obviously it is not.